jobs/check-aws-regions.Jenkinsfile

```diff
@@ -25,8 +25,9 @@ cosaPod(serviceAccount: "jenkins"){
         disabled_regions = shwrapCapture("ore aws list-regions --disabled")
     }
     if (disabled_regions != "None") {
+        echo "echo regions: ${disabled_regions}"
```

Comment:
pretty sure we should delete this line

Reply:
ah yea, i ll get rid of this. I was trying to test something.